lib/date_time/Kconfig

```diff
@@ -51,6 +51,19 @@ config DATE_TIME_NTP_QUERY_TIME_SECONDS
 	int "Duration in which the library will query for NTP time, in seconds"
 	default 5
 
+config DATE_TIME_RETRY_COUNT
+	int "Reattempt date-time updates at a reduced interval, up to this many times in a row"
+	default 0
+	help
+	  If this value is nonzero, failed date-time updates will be retried at a reduced interval,
+	  up to the number of times specified.
+
+config DATE_TIME_RETRY_INTERVAL_SECONDS
+	int "Time between date-time update retries, in seconds"
+	default 60
+	help
+	  The number of seconds to wait between date-time update retries.
+
```

Comment:
Nit: I'd move these before `config DATE_TIME_MODEM` right after `config DATE_TIME_TOO_OLD_SECONDS` just to have update interval and retry configs together as they are configuring the update logic.

Reply:
I'm not seeing this done although there is thumbs up.

Reply:
My bad!